recipes/logfault/all/conanfile.py

```diff
@@ -0,0 +1,91 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class PackageConan(ConanFile):
+    name = "logfault"
+    description = "Simple to use, header only C++ library for application-logging on all major platforms."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jgaa/logfault"
+    topics = ("logging", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+    generators = "CMakeToolchain"
```

Comment:
```suggestion
```

Duplicated with CMakeToolchain class usage.

Reply:
Are you sure? I removed it yesterday, and that broke the build. Adding it back fixed the build.

Reply:
Yes, I'm sure. At line [41](https://ci-conan-stg.jfrog.team/job/staging-v1/job/cci/view/change-requests/job/PR-1197/1/console) has CMakeToolchain declared already.